data/json/items/tool/science.json

```diff
@@ -170,5 +200,205 @@
+  },
+  {
+    "id": "microscope",
+    "type": "TOOL",
+    "category": "tools",
+    "name": "microscope",
+    "description": "A classic tool for the ages! This hefty, durable piece of lab equipment makes small things look bigger.  Without electricity the light doesn't work, but the rest of it is surprisingly functional.  Unfortunately it's not useful for very much at the moment.",
+    "weight": 2000,
+    "volume": "3800ml",
+    "price": 3000,
+    "to_hit": -5,
+    "material": [ "steel", "glass" ],
+    "symbol": "G",
+    "color": "white"
+  },
+  {
+    "id": "microscope_dissecting",
+    "type": "TOOL",
+    "category": "tools",
+    "name": "dissecting microscope",
+    "description": "This microscope has less magnification than a standard one.  It's also stereoscopic and produces a neat 3d image of the surface.  It was typically used for doing dissections and other fine detail work on small creatures.  It still works pretty well with an added light source, but what would you do with it?",
+    "weight": 2200,
+    "volume": "4000ml",
+    "price": 3000,
+    "to_hit": -5,
+    "material": [ "steel", "glass" ],
+    "symbol": "G",
+    "color": "white"
+  },
+  {
+    "id": "funnel_separation",
+    "type": "TOOL",
+    "category": "tools",
+    "name": "separation funnel",
+    "description": "This thing doesn't look much like a funnel.  It's a teardrop shaped piece of glass with a stopper at the wide end and a valve at the other.  It can be used to wash one liquid with another, like water and an oil, and then draw off whichever of the two liquids contains a higher amount of whatever compound you're trying to isolate.  With a high level of skill it can also be used to make juvenile farting noises.",
+    "weight": 200,
+    "volume": "700ml",
+    "price": 100,
+    "to_hit": -5,
+    "material": [ "glass", "plastic" ],
+    "looks_like": "flask_glass",
+    "symbol": "V",
+    "color": "light_blue",
+    "qualities": [ [ "SEPARATE", 1 ] ]
+  },
+  {
+    "id": "burette",
+    "type": "TOOL",
+    "category": "tools",
+    "name": "burette",
+    "description": "A tall glass column with a stopcock at the end.  If you're serious about chemistry, you probably have a tattoo of one of these somewhere.",
+    "weight": 300,
+    "volume": "1000ml",
+    "price": 1000,
+    "to_hit": -4,
+    "material": [ "glass", "plastic" ],
+    "looks_like": "glass_tube_small",
+    "symbol": "|",
+    "color": "light_blue"
+  },
+  {
+    "id": "rotovap",
+    "type": "TOOL",
+    "category": "tools",
+    "name": "rotary evapourator",
+    "description": "Just looking at this thing makes you feel like a proper mad scientist.  It's a series of glass tubes and round flasks, connected to a central motor and a heating element.  The heating element warms one flask, kept rotating by the motor, evapourating the contents.  The vapour is then condensed in the tubes and collected in another flask, in case you wanted to save it for later.",
```

Comment:
```suggestion
    "description": "Just looking at this thing makes you feel like a proper mad scientist.  It's a series of glass tubes and round flasks, connected to a central motor and a heating element.  The heating element warms one flask, kept rotating by the motor, evaporating the contents.  The vapor is then condensed in the tubes and collected in another flask, in case you wanted to save it for later.",
```